testinfra/modules/base.py

```diff
@@ -35,11 +35,11 @@ def run_expect(self, expected, command, *args, **kwargs):
         return out
 
     def run_test(self, command, *args, **kwargs):
-        """Run command and check it return an exit status of 0 or 1
+        """Run command and check it return an exit status of 0, 1 or 127
 
         :raises: AssertionError
         """
-        return self.run_expect([0, 1], command, *args, **kwargs)
+        return self.run_expect([0, 1, 127], command, *args, **kwargs)
```

Comment:
@philpep Yes that would be great.  Apologies, it has been a bad week at work so I've not had time to revisit this.